src/wordler.py

```
@@ -0,0 +1,111 @@
+    for clue in clues:
+        parts = clue.split('-')
+        letter = parts[0].lower()
+        position = int(parts[1])
+
+        if len(parts) == 2: # e.g. "w-1" or "e-0"
+            if 1 <= position <= 5:
+                known_positions[position - 1] = letter
+            elif position == 0:
+                present_letters.append(letter)
+        elif len(parts) == 3: # e.g. "e-0-1"
+            if position == 0:
```

Ruff (0.12.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)